src/rpcblockchain.cpp

```diff
@@ -174,7 +174,7 @@ UniValue getblockcount(const UniValue& params, bool fHelp)
     if (fHelp || params.size() != 0)
         throw runtime_error(
             "getblockcount\n"
-            "\nReturns the number of blocks in the longest block chain.\n"
+            "\nReturns the number of blocks in the longest blockchain.\n"
```

Comment:
We should have a style guide about this one. Personally I prefer "block chain", which is used in the Satoshi paper and in the Zcash protocol spec. Also, in this particular case "longest" is just plain wrong. The correct terminology is "best valid block chain" as defined in section 3.3 of the protocol spec.

Reply:
In the majority of cases in the source code, it is written blockchain. So the 2 cases I changed were the only ones I found that the words were separate. But I have no problem, I will change as per your preference.